packages/ui-components/style/index.css

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.elyra-expandableContainer-button {
+  background-color: transparent;
+  vertical-align: middle;
+  padding: 0;
+  width: 20px;
+}
+
+.elyra-expandableContainer-button:hover {
+  cursor: pointer;
+}
+
+.elyra-expandableContainer-actionButton:hover {
+  background-color: #ffffff;
+}
+
+.elyra-expandableContainer-actionButton:active {
+  background-color: #eeeeee;
+}
+
+.elyra-codeSnippetsHeader {
```

Comment:
we shouldn't have any css with  code snippet in the name in this package, it should either be moved by to code snippet's css file or renamed